src/devices/Seesaw/SeesawTouch.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+
+namespace Iot.Device.Seesaw
+{
+    public partial class Seesaw : IDisposable
+    {
+        /// <summary>
+        /// Reads the analog value on an capacitive touch-enabled pin.. 
+        /// </summary>
+        /// <param name="pinId">>The number of the pin to read.</param>
+        /// <returns>An analogue value betweeen 0 and 1023 that represents the capacitance read from the pin.</returns>
+        public ushort TouchRead(byte pinId)
+        {
+            if(!HasModule(Seesaw_Module.Touch))
```

Comment:
Ok, please fix the style in this PR, refer to https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

I'll re-review once this is fixed